AWS Lambda SDK: Report outcome error data with event instead of root span tags

medikoo opened the pull request.

Danwakeem commented on node/packages/aws-lambda-sdk/instrument/index.js:
```diff
@@ -138,27 +136,16 @@ const closeTrace = async (outcome, outcomeResult) => {
   };
 
   try {
+    const endTime = process.hrtime.bigint();
```
Just curious why the `responseStartTime` and `endTime` became one? 🤔

medikoo replied:
Good question, I should have probably split this into another commit.

`responseStartTime` was used to measure the overhead of the extension, and was measured around the same moment as `endTime`, so this is a simplification to rely on single timestamp (below now we write summary by subtracting `endTime` instead of `responseStartTime`)

Danwakeem reviewed (dismissed):
Looking good 👍

medikoo committed:
refactor: Remove `responseStartTime` in favor of `endTime`

medikoo committed:
refactor: Report outcome error data with event instead of root span tags

BREAKING CHANGE:
Outcome error meta is no longer provided with outcome error tags on `aws.lambda` root span.
Instead it's transported with captured event which shares same timestamp with `aws.lambda` root span end time
